### DISCUSSION TITLE
Is Spree fully translatable?

### DISCUSSION THREAD

**Discussion by demodes** [spree/spree#11719] [Q&A]
Hi, this project looks great and I was just curious if it is fully 100% translatable? I am asking because the demo 
( https://spree.vercel.app/ ) has language switcher in the footer, but after clicking on it, it doesn't change the language. All texts are the same. 

![spree-translating](https://user-images.githubusercontent.com/23063636/174435076-ac8d57ea-7c03-45a7-a8cc-99da810f5502.png)

**Comment by rahulsingh321** [comment]
Yes, Spree is fully translatable. Spree is a multi language eCommerce platform that uses **I18n** for translations. 
You can checkout spree demo [here](https://demo.spreecommerce.org),  that has live demo to easily switch languages. ![IMG-20220621-WA0002.jpg](https://user-images.githubusercontent.com/22743359/174728595-6d33e785-f700-4985-ab3e-b6e7be6c1244.jpg)

For more info: [Official Internationalisation Documentation](https://dev-docs.spreecommerce.org/customization/i18n)

**Comment by demodes** [comment]
Thank you for the information. It looks great. 
I tried the admin demo through heroku. Looks also good.
